tests/test_jitterbuffer.py

```diff
@@ -4,6 +4,13 @@
 from aiortc.rtp import RtpPacket
 
 
+async def send_rtcp_pli(media_ssrc: int) -> None:
+    """
+    Send an RTCP packet to report picture loss.
+    """
+    print("[INFO][JitterBuffer] PLI sent to media_ssrc:", media_ssrc)
```

Comment:
This print() statement seems kind of pointless. Could we instead define a function in each test which stores a value which we can assert?

Reply:
Well you're right that it does nothing currently - I just thought it looks better then a pass statement. I'm not sure if I get what you want to achieve with the assert method though. For me this would be also pointless, as the async function will be run after the test finishes and we can not run a synchronous method with asyncio.ensure_future() (jitterbufer.py line 44 an 54).

Reply:
My point is we are not asserting the function was actually called.